woodwork/schema_column.py

```diff
@@ -46,12 +46,13 @@ def _get_column_dict(name,
 
 
 def _validate_logical_type(logical_type):
-    ltype_class = _get_ltype_class(logical_type)
+    if logical_type is not None:
+        ltype_class = _get_ltype_class(logical_type)
```

Comment:
Wondering if it would be better to only call this function if there is a non-`None` logical type supplied in `_get_column_dict` instead of updating this to handle the `None` case? Is there an advantage to this approach?

Reply:
Hmm I guess I was thinking that we send null values to `_validate_accessor_params`, but even there we don't pass possibly null values to the actual `_check_time_index` type functions, just the larger validate function. I'm fine with pulling the check outside of the validate_logical_type function